nebari-dev/nebari
# Remove unused AWS terraform modules

## Reference Issues or PRs
Closes #2526 
Closes #2624 
Closes #2625 
Closes #2626  
Closes #2627  
Closes #2628  
Closes #2629  
Closes #2630 



## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [x] Other (please describe): Remove unused legacy code

## Testing


## Any other comments?

## How to test this PR?
1. Deploy Nebari to AWS
2. Make sure all services work properly
3. Destroy Nebari
